packages/api/test/e2e/staking.e2e.ts

```diff
@@ -281,27 +281,20 @@ describe('Staking Governance (Sudo Required)', () => {
   });
 
   test('Force unstake', async done => {
-    const bob_stash = keyring.addFromUri('//Bob//stash');
+    const bobStash = keyring.addFromUri('//Bob//stash');
     // bond bob's stash account.
     await api.tx.staking.bond(bob.address, 10_000, 'controller').signAndSend(
-      bob_stash,
+      bobStash,
       async ({ status }) => {
         if (status.isInBlock) {
-          const controller = (await api.query.staking.bonded(bob_stash.address)) as Option<AccountId>;
+          const controller = (await api.query.staking.bonded(bobStash.address)) as Option<AccountId>;
```

Comment:
would be great if we can avoid this nested pattern. It's hard to read, maintain, extend 💭